src/openMVG/robust_estimation/robust_estimator_LORansacKernelAdaptor.hpp

```diff
@@ -0,0 +1,119 @@
+#pragma once
+
+#include <openMVG/robust_estimation/robust_estimator_ACRansacKernelAdaptator.hpp>
+#include <openMVG/numeric/numeric.h>
+#include <openMVG/multiview/conditioning.hpp>
+
+namespace openMVG {
+namespace robust {
+
+
+template <typename SolverArg,
+  typename ErrorArg,
+  typename UnnormalizerArg,
+  typename ModelArg = Mat3,
+  typename SolverLSArg = SolverArg>
+class KernelAdaptorLoRansac : public ACKernelAdaptor<SolverArg, ErrorArg, UnnormalizerArg, ModelArg>
```

Comment:
Need doc with explanation for template arguments, in particular the fact that you need to different solvers for the estimation and for the refinement.